compliance/azure/azure_untagged_resources/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v2.13
+
+- Added the ability to check for key=value pairs
+- Added the ability to select a resource in the incident and enter a tag in key=value format to add to the specific resource
```

Comment:
Would keep these consistent with changelog for AWS counterpart here - https://github.com/flexera-public/policy_templates/pull/1178/files